src/serializer/ResidualHeapSerializer.js

```diff
@@ -980,7 +980,11 @@ export class ResidualHeapSerializer {
 
   _serializeValueSymbol(val: SymbolValue): BabelNodeExpression {
     let args = [];
-    if (val.$Description) args.push(t.stringLiteral(val.$Description));
+    if (val.$Description instanceof AbstractValue) {
```

Comment:
Yes I can add that to this pull request as well but I thought the idea was to separate this from serializing the symbols (PR #833)